Automattic/jetpack
# RNA buttons: add styles and storybook documentation

#### Changes proposed in this Pull Request:

* Add color variables.
* Add overriding styles for the `@wordpress/components` buttons.
* Added a documentation page in Storybook.

#### Jetpack product discussion

p6TEKc-5rF-p2

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions for Storybook:

* Go to the Storybook folder `cd projects/js-packages/storybook`
* Run `pnpm install`
* Run `pnpm run storybook:dev`
* Storybook should open in your browser.
* Go to Components → Buttons → Docs and play around with the various buttons.

#### Screenshot

![image](https://user-images.githubusercontent.com/390760/138317951-db1cf684-0e91-497d-9c0d-455510a78a57.png)
